restful/best-practices.md

```diff
@@ -232,27 +232,6 @@ During the deprecation timespan the responses of the deprected API SHOULD have a
 | 299       | Miscellaneous Persistent Warning | Section 5.5.7 |
 
 
-### Design APIs Conservatively
```

Comment:
I would not delete these best practices. If someone is really interested in writing good APIs it can help. Everyone else won't read it anyway. :)